app/root.css

```diff
@@ -498,7 +464,7 @@ svg {
 
 /* defines the standard left and right margins */
 .page-body {
-  padding: 0px 13.333vw;
+  padding: 0px var(--spacing-128);
```

Comment:
This makes me uncomfortable but I'll merge it anyway

Reply:
vw padding on clamped width means narrower content on larger screens... so the padding also needs to be clamped (though the "correct" value might need to be changed after design review on various screen sizes I guess .. I just picked a closest option to the 13vw)